src/privileged/blurts/FirefoxMonitor.jsm

```diff
@@ -197,7 +197,7 @@ function makeSpanWithLinks(aStrParts, doc) {
     anchor.setAttribute("style", "color: #0060DF");
     anchor.setAttribute("href", "javascript:void(0)");
     anchor.addEventListener("click", (event) => {
-      doc.defaultView.openUILinkIn(str.link, "tab");
+      doc.defaultView.openUILinkIn(str.link, "tab", {});
```

Comment:
I think we should use the null principal here (and elsewhere) - but this is fine for now if it works. I'll talk to johannh/jkt about the right principal to use here and we can add that in a later commit.